src/application.rs

```diff
@@ -143,6 +143,7 @@ impl Application {
                             state.connected_user(endpoint, &user);
                         }
                         NetMessage::UserMessage(content) => {
+                            // Note: can this unwrap actually fail?
```

Comment:
The only case this could happen is if the other `termchat` is a malicious `termchat` that talks without greets. I would ignore the message if the user is unknown.

Reply:
The rest of changes are ok 👍🏻